contracts/src/v0.8/KeeperRegistrar.sol

```diff
@@ -299,7 +299,14 @@ contract KeeperRegistrar is TypeAndVersionInterface, ConfirmedOwner, ERC677Recei
     address sender,
     uint256 amount,
     bytes calldata data
-  ) external onlyLINK permittedFunctionsForLINK(data) isActualAmount(amount, data) isActualSender(sender, data) {
+  )
+    external
+    override
```

Comment:
Requiring override is removed in 0.8.13 if a base contract has a concrete implementation. Downgrading reintroduces this requirement.